internal/cmd/plugin/install_utils.go

```diff
@@ -0,0 +1,134 @@
+package plugin
```

Comment:
Not a huge fan of files with the name "utils" in it... they can become black holes of random functionality. Can we either put these helper functions in the relevant command_*.go file or the root command.go file?